rotkehlchen/db/upgrades/v40_v41.py

```diff
@@ -204,6 +204,33 @@ def _reset_decoded_events(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _reset_decoded_events')
 
 
+def _remove_bittrex_data(write_cursor: 'DBCursor') -> None:
+    """
+    Removes bittrex settings and credentials from the DB
+    Code taken from v36->v37 upgrade from ftx.
+    """
+    log.debug('Enter _remove_bittrex_data')
+    write_cursor.execute(
+        'DELETE FROM user_credentials WHERE location=?',
+        (Location.BITTREX.serialize_for_db(),),
+    )
+    write_cursor.execute(
+        'DELETE FROM used_query_ranges WHERE name LIKE ? ESCAPE ?;',
+        (f'{Location.BITTREX!s}\\_%', '\\'),
+    )
+    non_syncing_exchanges_in_db = write_cursor.execute(
+        'SELECT value FROM settings WHERE name="non_syncing_exchanges"',
+    ).fetchone()
+    if non_syncing_exchanges_in_db is not None:
+        non_syncing_exchanges = json.loads(non_syncing_exchanges_in_db[0])
```

Comment:
should not happen, but since this is a DB ugprade and they tend to break really badly please handle the json decoding error. Handling should just ignore it and log an error.